docs/pages/home.js

```diff
@@ -0,0 +1,96 @@
+// @flow strict
+import { Box, Button, Flex, Heading, Text } from 'gestalt';
+import { type Node } from 'react';
+import HomePageCard from '../components/HomePageCard.js';
+import HomePageSection from '../components/HomePageSection.js';
+import Page from '../components/Page.js';
+import {
+  HowToWork,
+  Development,
+  Design,
+  A11Y,
+  Color,
+  DesignTokens,
+  Hero,
+} from '../graphics/HomePageIcons.js';
+
+export default function HomePage(): Node {
+  return (
+    <Page title="Welcome to Gestalt" isFullWidth>
+      <Box width="100%">
+        <Flex direction="column">
+          {/* Hero */}
+          <HomePageSection>
+            <Flex gap={8} alignItems="center">
+              <Flex gap={6} direction="column">
+                <Heading>Gestalt is Pinterest&rsquo;s design system</Heading>
+                <Text size="lg">
+                  We’re here to help you build experiences that inspire people to create the life
+                  they love
+                </Text>
+
+                <Button text="See what’s new" role="link" rel="nofollow" href="/whats_new" />
+              </Flex>
+              <Flex.Item minWidth="60%">
```

Comment:
@ayeshakmaz on mobile this results in a horizontal scroll bar - could we fix this? 
![image](https://user-images.githubusercontent.com/127199/156199945-8176c2d9-f0e2-4523-9821-dab08141591c.png)

Reply:
Fixed, chatted with Hector and we remove it on mobile